README.md

```diff
@@ -71,67 +71,133 @@ Running 2 tests for test/SignatureAccount.t.sol:SignatureAccountTest
-#  "params": [{"sender": "0xF19518B9424D8B0444b09E5B4631E728367caC20", "nonce": "2", "initCode": "0x", 
-#  "callData": ...}
-# ...
+# {
+#   "jsonrpc": "2.0",
+#   "id": 1,
+#   "method": "eth_sendUserOperation",
+#   "params": [
+#    ...
+#     },
+#     "0x0576a174D229E3cFA37253523E645A78A0C91B57"
+#   ]
+# }
+# 
+# ------------Sending payload to bundler--------------
+#
+# {
+#   "error": {
+#     "code": -32502,
+#     "data": "account uses banned opcode: SELFBALANCE",
+#     "message": "account uses banned opcode: SELFBALANCE"
+#   },
+#   "id": 1,
+#   "jsonrpc": "2.0"
+# }
 ```
 
-### Generate UserOp payload and send to bundler
-*(prerequisite: environment needs python3 installed to run below script)*
+### Interacting with Account accessing invalid Storage Slot
+The bundler should reject our request since we are not accessing the valid storage slot.
 ```bash
-# The private key here corresponds to the account owner address
-$ export PRIVATE_KEY=${PRIVATE_KEY_FOR_SIGNING_USER_OP}
-$ export RPC_URL=${GOERLI_RPC_ENDPOINT} 
-$ export ACCOUNT_ADDR=${4337_ACCOUNT_ADDRESS}
-$ export BUNDLER_URL=${BUNDLER_ENDPOINT} # may use stackup free endpoint here
+$ export PRIVATE_KEY=123456
+$ export ACCOUNT_ADDR=0xa729a76caadb6fdcD9c198cEd19b5D4e54bA0485
+$ export RPC_URL=${SEPOLIA_ENDPOINT}
+$ export BUNDLER_URL=${BUNDLER_ENDPOINT}
 
 # Run command at project root:
 $ ./bash/payload_builder.sh -a
 
 # Expected output:
-# 
+#
 # Generating userOperation...
 # Building userOp http payload for bundler...
-
+#
 # ------------Result Payload--------------
-# 
-# {"jsonrpc": "2.0", "id": 1, "method":eth_sendUserOperation
-# ...}
+#
+# {
+#   "jsonrpc": "2.0",
+#   "id": 1,
+#   "method": "eth_sendUserOperation",
+#   "params": [
+#    ...
+#     },
+#     "0x0576a174D229E3cFA37253523E645A78A0C91B57"
+#   ]
+# }
 # 
 # ------------Sending payload to bundler--------------
 #
-# {"id":1,"jsonrpc":"2.0","result":"0xd9fb9b74014af5...."}
+# {
+#   "error": {
+#     "code": -32502,
+#     "data": "account has forbidden read to 0x87224F6D41DF6044ddd30a87bBdEeBc8c8CAc4f0 slot 4dbb180290de92ae0711e87110c97f6daba9f11cdfc121096b461bdc56cfe39f",
+#     "message": "account has forbidden read to 0x87224F6D41DF6044ddd30a87bBdEeBc8c8CAc4f0 slot 4dbb180290de92ae0711e87110c97f6daba9f11cdfc121096b461bdc56cfe39f"
+#   },
+#   "id": 1,
+#   "jsonrpc": "2.0"
+# }
+```
+
+### Deploy a standard 4337 Account on Sepolia
+Try modifying `contracts/bundler/NonStandardAccount.sol` then deploy the account and play with it. You may get some Sepolia eth on https://sepolia-faucet.pk910.de or https://sepoliafaucet.com/
+
+Hint:
+
+1. Comment out codes that will likely cause failure at Bundler's validation stage.  
+2. (Optional) The `_validateSignature` function of `NonStandardAccount.sol` isn't properly implemented yet, it doesn't do any signature check. Add a verification logic in the function so only wallet owner can use the account. 
```

Comment:
No need to implement this here unless we are tasking them with implementing a standard account with knowledge from prior tasks.